src/provider/CoinbaseWalletProvider.ts

```diff
@@ -218,7 +245,7 @@ export class CoinbaseWalletProvider
 
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
-  public setProviderInfo(jsonRpcUrl: string, chainId: number) {
+  public setProviderInfo(jsonRpcUrl: string, chainId?: number) {
```

Comment:
chainId isn't used?

Reply:
Yeah, IDK why we still have it there. We could probably remove this